### DISCUSSION TITLE
Compatible Coral TPU Model Types

### DISCUSSION THREAD

**Discussion by Grego3j** [blakeblackshear/frigate#13819] [Ask A Question]
Hello Everyone, 

I am currently training models to run on Frigate using the Coral TPU for inference. So far, I have tried EfficientDet-Lite0 and SSD MobileNet V1, both of which are compatible with the TPU and with Frigate (although some modifications were necessary to run EfficientDet-Lite). My main question is will any SSD model that can be quantized and compiled for the Edge TPU run with minimal modifications on Frigate? Also, does anyone have any suggestions for better models to use? 

Thanks for the help!

**Comment by NickM-27** [comment]
> My main question is will any SSD model that can be quantized and compiled for the Edge TPU run with minimal modifications on Frigate? 

that's a very open ended question, in general SSD models should work. I believe Frigate+ uses SSDLite MobileDet
